reactioncommerce/reaction-next-starterkit
# feat: cart and mini-cart summary

Resolves #209 
Impact: **minor**
Type: **feature**

## Issue

Hook-up the cart and mini-cart summary components to real data from the GraphQL API.

## Solution

Hooked up the cart and mini-cart summary to data using the `@withCart` decorator

Reduced the number of car query/mutation files while simultaneously reducing the duplicated fields across those queries/mutations. Adding fields to 5 different files was very error prone. This refactor makes it a fair bit easier to deal with.

## Testing

1. Add items to the cart
2. Open the cart drawer and verify the total price of all goods
3. Open the cart page and verify the totals are correct.

**Note: there are no shipping or tax totals at the moment**
